ci-operator/step-registry/vsphere/deprovision/usertags/vsphere-deprovision-usertags-ref.yaml

```diff
@@ -0,0 +1,15 @@
+ref:
+  as: vsphere-deprovision-usertags
+  from: upi-installer
```

Comment:
The same comments as https://github.com/openshift/release/pull/52207/files#r1631020023